lib/Doctrine/DBAL/Migrations/Tools/Console/Command/MigrateCommand.php

```diff
@@ -75,25 +75,25 @@ protected function configure()
 
     <info>%command.full_name% --query-time</info>
 EOT
-        );
+            );
 
         parent::configure();
     }
 
     public function execute(InputInterface $input, OutputInterface $output)
     {
         $configuration = $this->getMigrationConfiguration($input, $output);
-        $migration = new Migration($configuration);
+        $migration     = new Migration($configuration);
```

Comment:
@Neirda24 We never align code or comment. This add useless noise in the history.